tools/rimage/config/lnl.toml

```diff
@@ -56,7 +56,7 @@ name = "ADSPFW"
 load_offset = "0x40000"
 
 [module]
-count = 25
+count = 26
```

Comment:
maybe this patch need on hold for a while and wait for: #8490 merge, after that, you only need add it to src/audio/igo_nr folder.
currently, this is duplicated.

Reply:
It's merged, the modularize patch is only for MTL.